docs/VnVPlan/VnVPlan.tex

```diff
@@ -158,18 +158,35 @@ \subsection{Verification and Validation Team}
 
 \subsection{SRS Verification Plan}
 
-\wss{List any approaches you intend to use for SRS verification.  This may
-  include ad hoc feedback from reviewers, like your classmates (like your
-primary reviewer), or you may plan for something more rigorous/systematic.}
-
-\wss{If you have a supervisor for the project, you shouldn't just say they will
-  read over the SRS.  You should explain your structured approach to the review.
-  Will you have a meeting?  What will you present?  What questions will you ask?
-  Will you give them instructions for a task-based inspection?  Will
-  you use your
-issue tracker?}
-
-\wss{Maybe create an SRS checklist?}
+To verify the SRS document, we will use the following methods:
+\begin{enumerate}
+  \item Formal reviews with our TA, Yiding. A checklist will be used to track
+    the status of the review and be used as an instrument to verify our SRS.
+    The checklist to be used is at the end of this section of the document.
+  \item Ad-hoc peer reviews tracked through GitHub issues. Throughout the
+    course we receive peer feedback on the quality of our SRS and VnV Plan. The
+    peer feedback provided is valuable input which we will action through issues
+    and pull requests as identified in the development plan.
+  \item Milestone feedback provided through Avenue will also be used to
+    improve the checklist which we use as our instrument to measure
+    SRS verification.
```

Comment:
we should maybe include a meeting with project supervisor to go over primary use cases? probably can't touch on anything more than that though because going into specific requirements would be a bit too much for a meeting with the supervisor

Reply:
I just don't really like the lack of mention of any validation with the stakeholder

Reply:
We could, although just for reference I include going through use cases with the City in the validation plan. Perhaps that is a better place for it to be? 

I was thinking our internal verification for the requirements wouldn't need the City, but then the validation to make sure it actually works for them does need the City. We would take their feedback from the validation process and if a requirement in the doc needs updating we do it based on the validation process.

Reply:
sure okay, we can leave it out